!GameTexts/NewTexts.xml

```diff
@@ -1281,7 +1262,7 @@
 			<Text>[COLOR_POSITIVE_TEXT]After Attack[ENDCOLOR] Defending Enemy Unit is marked. Marked unit has reduced [ICON_ARROW_RIGHT] Movement [COLOR_POSITIVE_TEXT]next turn[ENDCOLOR].</Text>
 		</Row>
 		<Row Tag="TXT_KEY_PROMOTION_UNIT_ETHIOPIA_MAIM_ON_UNIT">
-			<Text>Catching</Text>
+			<Text>Maimed</Text>
```

Comment:
We already have one dummy promotion called "Maimed". I was looking for some substitute. This promotions is used after attack but before lowering movement on unit. After lowering it changes to "Maimed".

Reply:
Could say "Hobbled", or since it only exists for 1 turn and is very subtle, you could add an Easter egg here. 

"Ouchies"
"Unit is covered in booboos, and has reduced movement speed  [COLOR_POSITIVE_TEXT]next turn[ENDCOLOR]"